requirements_dev.txt

```diff
@@ -167,22 +158,15 @@ inflection==0.5.1
     #   pytest-factoryboy
 iniconfig==2.0.0
     # via pytest
-ipython==8.11.0
+ipython==8.14.0
     # via -r requirements_dev.in
 itsdangerous==2.1.2
     # via flask
-itypes==1.2.0
-    # via
-    #   -r ./requirements.txt
-    #   coreapi
```

Comment:
Zijn deze dependencies voor ons obsolete geworden? Met andere woorden, gebruiken we deze niet meer?